packages/foam-cli/src/commands/janitor.ts

```diff
@@ -38,8 +40,12 @@ export default class Janitor extends Command {
     const { workspacePath = './' } = args;
 
     if (isValidDirectory(workspacePath)) {
-      const graph = (await bootstrap(createConfigFromFolders([workspacePath])))
-        .notes;
+      const config = createConfigFromFolders([workspacePath]);
+      const services: Services = {
+        logger: console,
+        dataStore: new FileDataStore(config),
+      };
+      const graph = (await bootstrap(config, services)).notes;
```

Comment:
the `bootstrap` could fail and would throw an exception, I think it's ok because the command couldn't work in any case (we could add a nicer error message, if that's what you are referring to, but in the end I wonder how much different it would be from the standard stack trace).
What was your concern specifically?

Reply:
nevermind, I actually just read your second comment :) 
I feel for now it's ok this way (this PR doesn't change anything from the pre-existing behavior).
At some point there will be also a more comprehensive approach to the CLI (probably on our next command, or based on the conversation around foam core/vscode/cli), and I would wait until then to make further changes here